students/13th/Gyuwon/project_westagram/user/views.py

```diff
@@ -0,0 +1,62 @@
+import json
+import re
+import jwt
+import bcrypt
+
+from django.db.models import Q
+from django.views     import View
+from django.http      import JsonResponse
+from .models          import User
+
+
+class SignUpView(View):
+    def post(self, request):
+
+        try:
+            data            = json.loads(request.body)
+            name            = data['name']
+            email           = data['email']
+            password        = data['password']
+            phone_number    = data['phone_number']
+
+            if User.objects.filter(Q(email=email) | Q(name=name) | Q(phone_number=phone_number)).exists():
+                return JsonResponse({"message": "Existing user."}, status=409)
+
+            elif re.match('^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$', email) == None:
```

Comment:
```suggestion
            elif not re.match('^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$', email):
```
`None` 은 Falsy 한 값이므로 굳이 `==` 으로 비교해주실 필요가 없이 not 으로 대체해주실 수 있겠네요 😀